### ISSUE TITLE
[question] Project license type

### ISSUE THREAD

**Issue by Hellseher** [karllark/dust_attenuation#56]
Hi,

I'm about to prepare your project to Guix, may you advise the license type you
would like to have been assigned as I did not find any mentioning of any kind it
uses.

Thanks,
Oleg

**Comment by karllark** [comment]
See the licence file in the licenses subdir.

**Comment by Hellseher** [comment]
Hi,

I try to identify the type of the license and may it be considered as FOSS one.

Based on these statements:

```
Copyright (c) 2018, Karl Gordon All rights reserved.
```

and

```
Neither the name of the Astropy Team nor the names of its contributors may be
used to endorse or promote products derived from this software without specific
prior written permission.
```

I can't make an assumption that it's FOSS license.

But if the provided license was copied from Astropy project as it is - it may
inherit BSD-3 type as Astropy uses.

May you clarify, please?

Thanks,
Oleg

**Comment by karllark** [comment]
Fixed with PR #57.

**Comment by Hellseher** [comment]
Thanks for the update.
I think the issue may be closed as resolved.
